<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>car_search</class>
 <widget class="QMainWindow" name="car_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>上一张</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>下一张</string>
    </property>
   </widget>
   <widget class="QComboBox" name="car_type">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>72</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Besturn X80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lubao</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Audi Q5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Audi Q3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Audi A7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Audi A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Exclle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>encore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Park Avenue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Enclave</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Regal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zhonghua Junjie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>车型</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="car_color">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>72</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>white</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>purple</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>champagne</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>silver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
